costajohnt/day3.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function() {
+      $('p').click(function() {
+        $('p').hide();
```

Comment:
I suspect you wanted to close the second function here.  Since I think you wanted the `$('.hover').click()` to be setup on document.ready